t/01-wasm/directives/008-flag_directive_wasmtime.t

```diff
@@ -87,7 +87,111 @@ qr/setting flag: "debug_info=off"/
 
 
 
-=== TEST 5: flag directive - wasmtime - consume_fuel - on
+=== TEST 5: flag directive - wasmtime - cache_config_load - missing file
```

Comment:
```suggestion
=== TEST 5: wasmtime cache_config - missing file
```

We can also rename the file name with `008-wasmtime_cache_config.t`

Reply:
Oops, good catch. 

> We can also rename the file name with 008-wasmtime_cache_config.t

008-flag_directive_wasmtime.t should stay as is since it has the rest of the wasmtime flag tests.  I added a new file for this directive, like we do for the other standalone directives.

Reply:
Didn't realize it was the same file; yes for a new one.